HelloWorld.java

```diff
@@ -36,5 +36,6 @@
 public class HelloWorld {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://docker.home.zuerker.org:9000/sonar/coding_rules#rule_key=squid%3AS1118)